index.js

```diff
@@ -40,7 +40,7 @@ function fastifyMultipart (fastify, options, done) {
 
     const req = this.req
 
-    const stream = new Busboy({ headers: req.headers })
+    const stream = new Busboy(Object.assign({ headers: req.headers }, options))
```

Comment:
@nigelhanlon I would prefer if we copied the options using an `Object.keys()` and `for`  loop instead. `Object.assign()`  is super slow and it will likely still be slow in Node 10 (see https://twitter.com/dan_abramov/status/980436488860196864).